tests/components/thumbnail-test.js

```diff
@@ -142,27 +144,4 @@ describe('Thumbnail', function () {
       expect(thumbnailTime.length).toBe(1);
     }
   });
```

Comment:
In general it seems that there were very few tests added considering the amount of code that was committed. Do we have decent coverage of the new scenarios that were added?

Reply:
We have 3 new functions in the new file thumbnailContainer.js. I test them in thumbnail-container-test.js . Also we check do we need to use thumbnails or thumbnailsCrousel in thumbnailContainer.js . I test it in thumbnail-container-test.js too. 

thumbnail-test.js is old test, and difference with old thumbnail.js one and new is only in new styles, which is tested in thumbnail-container-test.js